lib/helpers/config.rb

```diff
@@ -6,3 +6,7 @@ def get_config(file)
   STDERR.puts e.backtrace.inspect
   return nil
 end
+
+def get_repo_config(file)
+  get_config("blog content/master/#{file}")
```

Comment:
Nice! Pretty DRY, EXEPT while Multi branch support has not be added yet but if we wanted to in the future this might not be noticed quickly! (Not worry about trying to fix now)